src/obr/cli.py

```diff
@@ -57,6 +57,20 @@ def check_cli_operations(
     return True
 
 
+def is_valid_workspace(filters: list = []) -> bool:
```

Comment:
Please add a short docstring, also the name is a bit misleading, as this functions mostly checks if then number of jobs is non_zero